wikidict/lang/de/__init__.py

```diff
@@ -0,0 +1,54 @@
+"""Deutsh language."""
+
+# Regex to find the pronunciation
+pronunciation = r"{{Lautschrift\|([^}]+)}}"
+
+# Regex to find the gender
+gender = r",\s+{{([fmn]+)}}"
+
+# Float number separator
+float_separator = "."
```

Comment:
Could you tell me the good value to put here @Moonbase59 :pray:?

Reply:
We also now have a third official gender "d" (diverse), so gender should probably be `[fmnd]`. Don’t know if it’s used already in the Wiktionary.

Reply:
Good to know. I will let you update the regexp as an exercise :)